packages/workspace/src/command-line/workspace-schematic.ts

```diff
@@ -323,29 +416,36 @@ async function executeSchematic(
   }
 }
 
-function parseOptions(args: string[], outDir: string): { [k: string]: any } {
-  const schemaPath = path.join(outDir, args[0], 'schema.json');
-  let booleanProps = [];
-  if (fileExists(schemaPath)) {
-    const { properties } = readJsonFile(
-      path.join(outDir, args[0], 'schema.json')
-    );
-    if (properties) {
-      booleanProps = Object.keys(properties).filter(
-        (key) => properties[key].type === 'boolean'
-      );
-    }
-  }
-  return yargsParser(args, {
-    boolean: ['dryRun', 'listSchematics', 'interactive', ...booleanProps],
-    alias: {
-      dryRun: ['d'],
-      listSchematics: ['l'],
-    },
+function parseSchematicOptions(
+  schematicArgs: string[],
+  schematicBooleanArgs: string[]
+) {
+  return yargsParser(schematicArgs, {
+    boolean: ['dryRun', 'interactive', 'verbose', ...schematicBooleanArgs],
```

Comment:
Did you remove aliases?

Reply:
those aliases never worked for workspace-schematic (`-l`, `-d` )

Reply:
Just checked. They work for me.

Reply:
- so `-l` doesn't work with latest nx 9 (which we are currently at)  
- `-d` works on both 9 and 10 

will add both back.

thanks

Reply:
1. so I forgot that  `-l` is already covered within nx-commands.ts

![image](https://user-images.githubusercontent.com/1223799/99286482-3a391b00-2839-11eb-98ae-52a51cd9569e.png)

2. `-d` is back
![image](https://user-images.githubusercontent.com/1223799/99286570-56d55300-2839-11eb-95d0-a4a52a0a6919.png)